GVFS/GVFS.Service/Mac/NotificationHandler.cs

```diff
@@ -0,0 +1,27 @@
+﻿using GVFS.Common.NamedPipes;
+using GVFS.Common.Tracing;
+using System;
+
+namespace GVFS.Service.Handlers

```

Comment:
Nit: Namespace should match folder layout `GVFS.Service.Handlers` -> `GVFS.Service.Mac`